pkg/util/users/users.go

```diff
@@ -70,6 +75,25 @@ func (strategy DefaultUserSyncStrategy) ProduceSyncRequests(dbUsers spec.PgUserM
 		}
 	}
 
+	// No existing roles are deleted or stripped of role membership/flags
+	// but team roles will be renamed and denied from LOGIN
```

Comment:
What would happen to roles coming from the Teams API if the API becomes unavailable ?

Reply:
Good point. I thought, there will be an error and the sync will stop, but no - it continues only with additional members. I return the errors now.